charts/k8s-monitoring-v1/docs/examples/application-observability/metrics.alloy

```diff
@@ -116,12 +117,14 @@ otelcol.processor.transform "default" {
       "set(attributes[\"namespace\"], attributes[\"k8s.namespace.name\"])",
```

Comment:
Setting `pod` and `namespace` as log resource attributes is very likely to no longer relevant if we send OTLP logs to the Grafana Cloud OTLP Endpoint and adopt the new Loki format for OTLP logs.
`pod` and `namespace` will become log metadata which requires a different LogQL query.
This comment applies to all the files where this config block is added.